code/game/objects/effects/landmarks.dm

```diff
@@ -306,14 +306,14 @@ INITIALIZE_IMMEDIATE(/obj/effect/landmark/start/new_player)
 	name = "Observer-Start"
 	icon_state = "observer_start"
 
-//xenos, morphs and nightmares spawn here
-/obj/effect/landmark/xeno_spawn
-	name = "xeno_spawn"
+//generic maintenance locations
+/obj/effect/landmark/generic_maintenance_landmark
+	name = "generic_maintenance_spawn"
 	icon_state = "xeno_spawn"
 
-/obj/effect/landmark/xeno_spawn/Initialize(mapload)
+/obj/effect/landmark/generic_maintenance_landmark/Initialize(mapload)
 	..()
```

Comment:
```suggestion
	. = ..()
```

no reason to not do it like this especially considering we're just doing a simple list addition.

Reply:
The reason not to do it is that we are explicitly returning a qdel, and we *want* this to runtime if we can't get that far

Reply:
ah wasn't aware of that